electron/app/components/Sample.tsx

```diff
@@ -89,9 +93,25 @@ const Sample = ({
         thumbnail={true}
         activeLabels={activeLabels}
         {...eventHandlers}
+        filter={filter}
       />
       <div className="sample-info" {...eventHandlers}>
-        {Object.keys(sample).sort().map(renderLabel)}
+        {Object.keys(sample)
+          .sort()
+          .reduce((acc, name) => {
+            const label = sample[name];
+            if (label && label._cls === "Classifications") {
+              return [
+                ...acc,
+                ...label[label._cls.toLowerCase()].map((l) => ({
+                  name,
+                  label: l,
+                })),
+              ];
+            }
+            return [...acc, { name, label }];
```

Comment:
I'm seeing duplicates of `Classifications` when selecting/deselecting any labels. This might need a `key` field too.
![image](https://user-images.githubusercontent.com/3719547/91356890-5db27380-e7be-11ea-9fe4-0ca8b1f36d4d.png)

Reply:
Ok, @brimoor this is what I believe you requested. Show each classification in a separate bubble for `Classifications`?

Reply:
The issue is that it should only show "sunny" and "outdoor" once. I got into this state after unchecking other labels.